### DISCUSSION TITLE
Is this how you load Spark?

### DISCUSSION THREAD

**Discussion by hugovdev** [Minestom/Minestom#1530] [Help] [answered]
I just downloaded the spark plugin for Minestom, but I have no clue on how to load it. I tried this, but is giving me an error:
![image](https://user-images.githubusercontent.com/23460704/203122309-d8d42327-6022-4b80-ba37-87907f32298f.png)

This is the error I'm getting:
![image](https://user-images.githubusercontent.com/23460704/203122674-f8e54c2d-079c-470d-9f90-e9f9b491477e.png)

**Comment by hugovdev** [comment]
Fixed that problem but now I get this:
![image](https://user-images.githubusercontent.com/23460704/203127182-fffaf749-fa59-47dc-908e-aba2a0410124.png)

**Reply by ZakShearman** [reply]
What minestom server distro/implementation (e.g Sabre) are you using?

**Reply by hugovdev** [reply]
Using the latest commit of this repo

**Reply by ZakShearman** [reply]
Oh I see getLogger has been changed to return ComponentLogger instead of org.slf4j.Logger. I would recommend making a PR to Spark to update their Minestom support if possible :)

**Comment by literallyfiro** [comment, marked as answer]
I made a PR to fix the spark loading issue. You can now download the latest version and it will work just fine

**Reply by hugovdev** [reply]
Thanks! :)
